### ISSUE TITLE
identitystore: missing `Active` attribute using `ListUsers`

### ISSUE THREAD

**Issue by optplx** [aws/aws-sdk-go-v2#2320]
### Describe the bug

I can't find a way to determine if a SSO user is active or not

### Expected Behavior

In the `User` struct of the `github.com/aws/aws-sdk-go-v2/service/identitystore` package, I expected to find a boolean for the `Active` attribute. 

In the console, there is a call to https://up.sso.eu-west-1.amazonaws.com/identitystore/ returning a list of users with this boolean for each user

```
{
    "Active": true,
    "Meta": {
        "CreatedAt": 1.1234567781234E9,
        "CreatedBy": "xxxxx",
        "UpdatedAt": 1.1234567781234E9,
        "UpdatedBy": "xxxxx"
    },
    "UserAttributes": {
        "emails": {
            "ComplexListValue": [
....
```

### Current Behavior

Currently I can't find any way to check this attribute with this SDK (I am maybe missing something here)

### Reproduction Steps

using this snippet, line 13 would not work
```go
func listUsers(ctx context.Context, idstore identitystore.ListUsersAPIClient, storeID string) {
	p := identitystore.NewListUsersPaginator(
		idstore, &identitystore.ListUsersInput{
			IdentityStoreId: aws.String(storeID),
		})

	for p.HasMorePages() {
		output, err := p.NextPage(ctx)
		if err != nil {
			panic(err)
		}
		for _, user := range output.Users {
			fmt.Println(user.Active) // This won't work
		}
	}
}
```

### Possible Solution

If possible, marshall this attribute in the json struct. I don't know the codebase well but I can have a look if you give me some pointers here

### Additional Information/Context

willing to contribute if this helps

### AWS Go SDK V2 Module Versions Used

v1.18.3

### Compiler and Version used

go version go1.19.3

### Operating System and version

darwin/amd64

**Comment by lucix-aws** [comment]
AWS SDKs are generated from models owned by service teams. In effect, the list of supported operations and their inputs/outputs (including how response fields are deserialized) are moderated externally to the SDKs.

Console- and internal- only operations/fields exist, I'm not familiar with these APIs but that is potentially the cause of the missing field here.

**Comment by RanVaknin** [comment]
Hi @optplx ,

Just to add to what @lucix-aws have mentioned. From the looks of it, the https://up.sso.eu-west-1.amazonaws.com/identitystore/ is an endpoint owned by the SSO (IAM identity center) not the IdentityStore team. Additionally, after digging internally I can see a model that is somewhat corresponds with this shape you shared here:
```
{
    "Active": true,
    "Meta": {
        "CreatedAt": 1.1234567781234E9,
        "CreatedBy": "xxxxx",
        "UpdatedAt": 1.1234567781234E9,
        "UpdatedBy": "xxxxx"
    },
    "UserAttributes": {
        "emails": {
            "ComplexListValue": [
....
```

Unfortunately this model is not available under the IdentityStore namespace and even if the IdentityStore team updates their model, that particular team does not have service side support. 
In other words, that endpoint `https://up.sso.eu-west-1.amazonaws.com/identitystore/` is likely doing some fancy weight lifting behind the scene aggregating data from multiple APIs.

This will have to be a feature request opened with the SSO / identityStore service teams. You can try to use AWS console to open a support ticket and file this feature request. 

Since this is not actionable by the SDK team, Im inclined to close this. If you decide to open the feature request please refer the support person to this thread. 

All the best,
Ran~
